envs/Jenkinsfile

```diff
@@ -33,7 +33,7 @@ psl.runsHere('production')
 
 @Field final String DOCKER_REGISTRY  = 'zrh-dreg-sp-1.esri.com'
 @Field final String DOCKER_PROJECT   = "${DOCKER_REGISTRY}/pyprt/"
-@Field final String DOCKER_IMAGE_REV = 'v9' // target revision of the generated images
+@Field final String DOCKER_IMAGE_REV = 'v10' // target revision of the generated images
```

Comment:
I'm not sure but don't we need to update https://github.com/Esri/pyprt/blob/a959d140fc18c26682d5ee1216ba51f7815faa3e/Jenkinsfile#L41C41-L41C44 as well?